apps/server/src/utils/__tests__/fileManagement.test.ts

```diff
@@ -90,3 +95,25 @@ describe('generateUniqueFileName', () => {
     expect(uniqueFilename).toBe(expectedFilename);
   });
 });
+
+describe('file index', () => {
+  it('sets index to 1 when there is no index', () => {
+    expect(incrementProjectNumber('test file.json')).toBe('test file (1).json');
+  });
+
+  it('increments to 2 when index is 1', () => {
+    expect(incrementProjectNumber('test file (1).json')).toBe('test file (2).json');
+  });
+
+  it('dose not count number not warped in parenthesise', () => {
+    expect(incrementProjectNumber('test file 1.json')).toBe('test file 1 (1).json');
+  });
+
+  it('dose not count number if there is not a space', () => {
```

Comment:
Question: is there a reason we want to make this affordance in code? It seems like the logic could be simpler if we decided not to care

```suggestion
  it('does not count number if there is not a space', () => {
```